pkg/compactor/block_upload.go

```diff
@@ -40,6 +42,10 @@ const (
 	validationHeartbeatTimeout  = 5 * time.Minute       // Maximum duration of time to wait until a validation is able to be restarted
 )
 
+var maxBlockUploadSizeBytesFormat = globalerror.CompactorBlockUploadMaxBlockSizeBytes.MessageWithPerTenantLimitConfig(
```

Comment:
No other error reported during block upload is currently using this way of reporting errors to user. I would suggest to tackle converting all block upload errors to the new system in a separate PR, and use "simple" way of error reporting in this PR.

Reply:
Changed to the simple way.